### PR TITLE
Bdd tests bmi qvq validation

### PR DESCRIPTION
**Pull request by fayaz0211**
Added QvQ validation tests for BMI Surveys

### CODE REVIEW

File: tests/acceptance-tests/base/utilities.py [hunk 1]
```diff
@@ -5,3 +5,22 @@ def convert_blank_data_to_empty_string(data):
         if data.lower() == 'blank':
             data = ''
         return data
+
+    @staticmethod
+    def replace_blank_with_empty_string(text):
```
Comment (dllmorgan) [inline tests/acceptance-tests/base/utilities.py:10]: This isn't an empty string. It's a single character space.

File: tests/acceptance-tests/base/utilities.py [hunk 2]
```diff
@@ -5,3 +5,22 @@ def convert_blank_data_to_empty_string(data):
...
+        return text.replace('<Blank>', ' ')
+
+    @staticmethod
+    def get_values_as_a_list(values):
+        new_values = values.split(',')
+        list_values = []
+        for new_val in new_values:
+            list_values.append(new_val)
+        return list_values
```
Comment (dllmorgan) [inline tests/acceptance-tests/base/utilities.py:19]: Isn't this the same as:

```
def get_values_as_a_list(values):
    return values.split(',')
```

Comment (fayaz0211) [reply tests/acceptance-tests/base/utilities.py:19]: Yes you are right.forgot to remove the loop.

File: tests/acceptance-tests/base/utilities.py (hunk 1 above)
Comment (fayaz0211) [reply tests/acceptance-tests/base/utilities.py:10]: Yes, got that wrong will change it.

**Pull request merged**